gov_uk_dashboards/axes/calculate_axis_range.py

```diff
@@ -0,0 +1,15 @@
+"""Function for calculating the axis range"""
```

Comment:
With the function defined as it is now, I _think_ the import would be
```
from gov_uk_dashboards.axes.calculate_axis_range import calculate_axis_range
```

where as if we moved the code into a file `gov_uk_dashboards/axes.py` the import would become

```
from gov_uk_dashboards.axes import calculate_axis_range
```

which I think looks a little nicer, and follows the convention with the colours.